rclcpp/src/rclcpp/generic_subscription.cpp

```diff
@@ -46,9 +48,20 @@ void GenericSubscription::handle_message(
 void
 GenericSubscription::handle_serialized_message(
   const std::shared_ptr<rclcpp::SerializedMessage> & message,
-  const rclcpp::MessageInfo &)
+  const rclcpp::MessageInfo & message_info)
 {
+    auto callback_ptr = static_cast<const void *>(&callback_);
+    auto rmw_info = message_info.get_rmw_message_info();
+    auto source_timestamp = rmw_info.source_timestamp;
+    TRACEPOINT(
+      dispatch_subscription_callback,
+      message.get(),
+      callback_ptr,
+      source_timestamp,
+      static_cast<const uint64_t>(rclcpp::TimeStamp<rclcpp::SerializedMessage>::value(*message).second));
```

Comment:
I think `rclcpp::TimeStamp<rclcpp::SerializedMessage>::value(*message)` does not work and its value is always 0. Can you check if it is actually 0 in some test case? If so, I suggest replacing it with 0.

```suggestion
      0);
```

Reply:
Thanks @isp-uetsuki .
I confirmed message timestamp is set to 0 on some  test cases.
I will accept your suggested modifications, after checking the related source code..

Reply:
@isp-uetsuki 
I have confirmed that the previous implementation was incorrect.
Also, since serialized messages have no header, I decided to fill the timestamps with zeros.
Thanks for your review.